x-pack/test/api_integration/apis/index.js

```diff
@@ -6,13 +6,22 @@
 
 export default function ({ loadTestFile }) {
   describe('apis', () => {
-    loadTestFile(require.resolve('./es'));
-    loadTestFile(require.resolve('./security'));
-    loadTestFile(require.resolve('./monitoring'));
-    loadTestFile(require.resolve('./xpack_main'));
-    loadTestFile(require.resolve('./logstash'));
-    loadTestFile(require.resolve('./kibana'));
+    // loadTestFile(require.resolve('./es'));
```

Comment:
consider adding a `This should not be merged into master...` comment (like the others) at the start of this section